exercises/pig-latin/description.md

```diff
@@ -5,10 +5,10 @@ confusing. It obeys a few simple rules (below), but when it's spoken
 quickly it's really difficult for non-children (and non-native speakers)
 to understand.
 
-- **Rule 1**: If a word begins with a vowel sound, add an "ay" sound to
-  the end of the word.
-- **Rule 2**: If a word begins with a consonant sound, move it to the
-  end of the word, and then add an "ay" sound to the end of the word.
+- **Rule 1**: If a word begins with a vowel sound, add an "ay" sound to the end of the word. Treat "xr" and "yt" at the beginning of a word as vowels (e.g. "xray" -> "xrayay", "yttria" -> "yttriaay").
```

Comment:
The inconsistency between talking about a vowel sound versus vowels causes me, a person reading this rule, to wonder why the distinction is necessary.

Very specifically: The second sentence tells me to treat `xr` and `yt` as **vowels**. However, this is insufficient information to determine whether the first sentence ("If a word begins with **a vowel sound**") applies.

If the first sentence reads "If a word begins **a vowel**, ...") XOR if the second sentence were to say "xr and yt at the beginning of a word make **vowel sounds**", then the terminology would be consistent and it would be clear that the first sentence is applicable to the second.

Reply:
I agree, talking about a vowel sound makes more sense. Thanks for the feedback.